app/features/TransactionSlice.ts

```diff
@@ -85,38 +137,40 @@ const forceLock = new Mutex();
  * @param account the account to get new transactions for
  * @param transactions the current transactions in the state, assumed to be sorted descendingly
  * @param limit the maximum number of transactions to ask the wallet proxy for
+ * @param loadShielded whether the transactions that should be retrieved are shielded transactions or not
```

Comment:
This name / comment is a bit misleading, I think it should be more clear, that it _only_ shielded transactions.

Reply:
I have change all of them to something prefixed with `only`.